ui/webui/test/helpers/installer.py

```diff
@@ -16,51 +16,55 @@
 # along with this program; If not, see <http://www.gnu.org/licenses/>.
 
 
-class Installer():
-    welcome_id = "installation-language"
-    storage_id = "installation-destination"
-    review_id = "installation-review"
-    progress_id = "installation-progress"
-    steps = [welcome_id, storage_id, review_id, progress_id]
+class InstallerSteps():
+    welcome = "installation-language"
+    storage = "installation-destination"
+    review = "installation-review"
+    progress = "installation-progress"
+    all = [welcome, storage, review, progress]
+
 
+class Installer():
     def __init__(self, browser, machine):
         self.browser = browser
         self.machine = machine
+        self.steps = InstallerSteps()
+        self.steps_list = self.steps.all
```

Comment:
I think it's redundant to have `InstallerSteps.all` and copy it again on the parent as `steps_list`. Why not just do `self.steps.all`?

Reply:
Or you can make `InstallerSteps` iterable. Then `steps` will work the same way as `steps_list`. The current solution is very confusing.